templates/service-template.yml

```
@@ -372,7 +372,11 @@ parameters:
   displayName: MAS SSO REALM for OSD Cluster Identity Provider Clients
   description: MAS SSO realm for OSD Cluster Identity Provider Clients
   value: "interim-test-realm"
-
+  
+- name: SELECT_SSO_PROVIDER
```

```suggestion
- name: SSO_PROVIDER_TYPE
```

`SSO_PROVIDER_TYPE` maybe?